deploy/common/k8sbased.sh

```diff
@@ -619,7 +638,7 @@ function launch_sensor {
       if [[ -f "$k8s_dir/sensor-deploy/chart/feature-flag-values.yaml" ]]; then
         helm_args+=(-f "$k8s_dir/sensor-deploy/chart/feature-flag-values.yaml")
       fi
-      if [[ "$sensor_namespace" != "stackrox" ]]; then
+      if [[ "${namespace}" != "stackrox" ]]; then
```

Comment:
why can't we stick to sensor_namespace in this function

Reply:
Previously there was a new local variable named `sensor_namespace` created [here](https://github.com/stackrox/stackrox/pull/9402/files/f1de2209f6fdfd5887a538aa9b8e88302976e2c7#diff-4b1235ef4151df5370f2051d5f312a8dd0e43faffeda4dcd93dc42f2fadb6a0bL587).
I have removed that one, since I think that it isn't required and instead would be confusing to have yet another namespace variable.

In the beginning of the function I have introduced a variable `namespace` (instead of `sensor_namespace`), because this is the `launch_sensor` function and it would be kinda implicit that `namespace` without any prefixing would mean `namespace for sensor`. Also see https://github.com/stackrox/stackrox/pull/9402#issuecomment-1895748655.

Reply:
Outdated by now, can we resolve?